web/ViewModels/ImageListViewModel.cs

```diff
@@ -3,5 +3,17 @@
 public class ImageListViewModel
 {
     public int PropertyId { get; set; }
-    public List<string>? FileList { get; set; }
+    public List<ImageFile>? FileList { get; set; }
+}
+
+public class ImageFile

```

Comment:
Can you rename this, maybe something like `ImageFileUrlModel`? When I see `ImageFile` I assumed it would include the image itself!